src/app/helptext/task-calendar/replication/replication.ts

```diff
@@ -2,234 +2,243 @@ import { T } from '../../../translate-marker';
- replication task. Choose from configured\
- <a href="--docurl--/tasks.html#periodic-snapshot-tasks"\
- target="_blank">Periodic Snapshot Tasks</a>. This replication task must\
- have the same <b>Recursive</b> and <b>Exclude Child Datasets</b> values\
- as the chosen periodic snapshot task. Selecting a periodic snapshot\
- schedule hides the <b>Schedule</b> field.'),
+    periodic_snapshot_tasks_tooltip: T('Snapshot schedule for this \
+ replication task. Choose from configured \
+ <a href="--docurl--/tasks.html#periodic-snapshot-tasks" \
+ target="_blank">Periodic Snapshot Tasks</a>. This replication task must \
+ have the same <b>Recursive</b> and <b>Exclude Child Datasets</b> values \
+ as the chosen periodic snapshot task. Selecting a periodic snapshot \
+ schedule removes the <b>Schedule</b> field.'),
 
     naming_schema_placeholder: T('Naming Schema'),
-    naming_schema_tooltip: T(''),
+    naming_schema_tooltip: T('Custom snapshots to be replicated. Enter \
+ the name and <a href="https://www.freebsd.org/cgi/man.cgi?query=strftime" \
+ target="_blank">strftime(3)</a> <i>%Y</i>, <i>%m</i>, <i>%d</i>, \
+ <i>%H</i>, and <i>%M</i> strings that match the snapshots to include in \
+ the replication.'),
 
     also_include_naming_schema_placeholder: T('Also Include Naming Schema'),
-    also_include_naming_schema_tooltip: T('Additional snapshots to\
- include in the replication with the periodic snapshot schedule. Enter\
- the <a href="https://www.freebsd.org/cgi/man.cgi?query=strftime"\
- target="_blank">strftime(3)</a> strings that match the snapshots to\
- include in the replication.<br><br>\
- When a periodic snapshot is not linked to the replication, enter the\
- naming schema for manually created snapshots. Has the same <i>%Y</i>,\
- <i>%m</i>, <i>%d</i>, <i>%H</i>, and <i>%M</i> string requirements as\
- the <b>Naming Schema</b> in a\
- <a href="--docurl--/tasks.html#periodic-snapshot-tasks"\
+    also_include_naming_schema_tooltip: T('Additional snapshots to \
+ include in the replication with the periodic snapshot schedule. Enter \
+ the <a href="https://www.freebsd.org/cgi/man.cgi?query=strftime" \
+ target="_blank">strftime(3)</a> strings that match the snapshots to \
+ include in the replication.<br><br> \
+ When a periodic snapshot is not linked to the replication, enter the \
+ naming schema for manually created snapshots. Has the same <i>%Y</i>, \
+ <i>%m</i>, <i>%d</i>, <i>%H</i>, and <i>%M</i> string requirements as \
+ the <b>Naming Schema</b> in a \
+ <a href="--docurl--/tasks.html#periodic-snapshot-tasks" \
  target="_blank">periodic snapshot task.'),
 
     auto_placeholder: T('Run Automatically'),
-    auto_tooltip: T('Set to either start this replication task\
- immediately after the linked periodic snapshot task completes or see\
+    auto_tooltip: T('Set to either start this replication task \
+ immediately after the linked periodic snapshot task completes or see \
```

Comment:
"or see" is not clear whether setting this shows options or the reader is supposed to go look at them.